gen3/bin/kube-setup-hatchery.sh

```diff
@@ -22,10 +22,10 @@ gen3 jupyter j-namespace setup
 
 
 # cron job to distribute licenses if using Stata workspaces
-if [ "$(g3kubectl get configmaps/manifest-hatchery -o yaml | grep "\"image\": .*stata.*")" ];
-then
-    gen3 job cron distribute-licenses '* * * * *'
-fi
+# if [ "$(g3kubectl get configmaps/manifest-hatchery -o yaml | grep "\"image\": .*stata.*")" ];
+# then
+#     gen3 job cron distribute-licenses '* * * * *'
```

Comment:
I think we still need this job?

Reply:
I commented it out thinking it was never finished, but if it's actually in use yeah for sure let's keep it!